tests/unit_tests/conftest.py

```diff
@@ -32,3 +34,10 @@ def __init__(self, *args, **kwargs):
             super().__init__(*args, **kwargs)
 
     monkeypatch.setattr("ert.cli.main.EvaluatorServerConfig", MockESConfig)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def ensure_bin_in_path():
```

Comment:
```suggestion
def ensure_bin_in_path():
    """
    Running pytest directly without enabling a virtualenv is perfectly valid.
    However, our tests assume that `job_dispatch.py` is in PATH which it may not be.
    This fixture prepends the path to the current Python for tests to pass when not in a virtualenv.
    """
```